src/main/java/com/revature/shms/models/Reservation.java

```diff
@@ -0,0 +1,31 @@
+package com.revature.shms.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+/**
+ * Reservation model
+ */
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    int reservationId;
+
+    @Column
+    String userId;
+
+    @Column
+    String status;
+
```

Comment:
Before merging, add in a one to many relationship. This way we don't have to merge these branches agian